src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java

```diff
@@ -94,5 +93,15 @@ private void runLoad(BulkLoadFileHistory history, String dataProvider, ParalogyI
 		ph.finishProcess();
 
 	}
+	
+	public APIResponse runLoad(String dataProvider, ParalogyIngestFmsDTO paralogyData) {
```

Comment:
We might not need this method as we can use the super class one.... maybe we need it due to having to deal with the "Pair" of genes.

Reply:
Yes, I believe we need to have this method as we have overridden the base `runLoad` method in order to deal with "Pair" of genes.

Reply:
I can exclude this for now, but for the following tickets, I would need this method as its used by the test cases.

Reply:
Ya... you are right sorry.